quiche/src/lib.rs

```diff
@@ -799,6 +802,7 @@ pub struct Config {
     hystart: bool,
 
     pacing: bool,
+    /// Send rate limit in Mbps
     max_pacing_rate: Option<u64>,
```

Comment:
This seems to be used [here] via RecoveryConfig (https://github.com/cloudflare/quiche/blob/master/quiche/src/recovery/congestion/pacer.rs#L73) and be bytes/sec

Reply:
I need to trace through this.  Shouldn't be blocking since it's just a comment.